File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int l=1;
        int r=1;
        int n = nums.size();
        while(r<n) {
            if(nums[r]==nums[r-1]) {r++; continue;} else { nums[l]=nums[r];l++;r++;}
        }
        return l;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i,j=1,1
        n = len(nums)
        if n<=1: return n
        while j<n:
            if nums[j]==nums[j-1]:
                j+=1
            else:
                nums[i] = nums[j]
                j+=1
                i+=1
        return i

        